Automattic/woocommerce-payments
# Return correct product prices datatype in WCPay

Fixes #2827 

#### Changes proposed in this Pull Request

Return prices as a string according to the convention with WC Core described in https://github.com/Automattic/woocommerce-payments/issues/2827.

Filters actions that should return prices as string:
- `woocommerce_product_get_price`
- `woocommerce_product_get_regular_price`
- `woocommerce_product_get_sale_price`
- `woocommerce_product_variation_get_price`
- `woocommerce_product_variation_get_regular_price`
- `woocommerce_product_variation_get_sale_price`
- `woocommerce_variation_prices`









#### Testing instructions





* As a shopper go to the Shop page, switch the currency to EUR.
* Add simple product to cart.
* Add variable product to cart.
* Add sale product to cart.
* Switch currencies to JPY and USD and see that the prices are shown correctly.
* Checkout and Pay, should be good without errors.

-------------------


**Post merge**
